CampaignConfig/mdc2020_trkana.cfg

```diff
@@ -0,0 +1,33 @@
+[global]
+includes = /mu2e/app/users/sophie/ProductionManager/Production/CampaignConfig/mdc2020_prolog.cfg
+release = MDC2020
+desc = MDC2020s
+release_v_i = r
+release_v_o = s
+db_version = v1_0
+db_purpose = perfect
+primary_name = IPAMuminusFlateMinus
+trkana_version_counter = a
+trkana_version = v01_01_00
+beam = 1BB
+email-to = sophie@fnal.gov
+[stage_trkana]
+job_setup.source_2 = /cvmfs/mu2e.opensciencegrid.org/Musings/TrkAna/%(trkana_version)s/setup.sh
+global.desc = %(release)s%(release_v_o)s_%(db_purpose)s_%(db_version)s_std_%(trkana_version)s
+global.stage_name = %(primary_name)sMix%(beam)s
+job_output_2.addoutput = *.%(stage_name)s.*.tka
+job_output_2.add_to_dataset = nts.%(submitter)s.%(stage_name)s.%(desc)s.tka
+job_output_2.dest = %(outdir_nts_tape)s/%(stage_name)s/%(desc)s/tka
+global.artRoot_dataset = nts.%(submitter)s.%(stage_name)s.%(desc)s.tka
+submit.n_files_per_job = 1
```

Comment:
I think TrkAna can use a much larger merge factor than 1.  Please estimate a number based on an output file size of ~1GByte.

Reply:
Will check